packages/react/fast-components-react-base/src/data-grid/data-grid-row.schema.2.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Complies with FAST Tooling 2.0
+ */
+export default {
+    $schema: "http://json-schema.org/schema#",
+    title: "Data grid row",
+    description: "A data grid row's schema definition.",
+    type: "object",
+    id: "@microsoft/fast-components-react-base/data-grid-row",
+    formPluginId: "@microsoft/fast-components-react-base/data-grid-row",
+    properties: {
+        rowData: {
+            title: "Row data",
+            type: "object",
+        },
+        gridTemplateColumns: {
+            title: "Grid template columns",
+            type: "string",
+        },
+        rowIndex: {
+            title: "Row index",
+            type: "number",
+        },
+    },
+    required: ["rowData", "gridTemplateColumns", "gridRowTop", "rowIndex"],
```

Comment:
I don't see gridRowTop enumerated in the properties

Reply:
removed.